ui/address/blocksValidated/AddressBlocksValidatedListItem.tsx

```diff
@@ -30,9 +29,13 @@ const AddressBlocksValidatedListItem = (props: Props) => {
           noIcon
           fontWeight={ 700 }
         />
-        <Skeleton isLoaded={ !props.isLoading } color="text_secondary" display="inline-block">
-          <span>{ timeAgo }</span>
-        </Skeleton>
+        <TimeAgoWithTooltip
+          timestamp={ props.timestamp }
+          enableIncrement={ props.page === 1 }
+          isLoading={ props.isLoading }
+          color="text_secondary"
```

Comment:
Seems like it is the default color for this type of info + property `display: inline-block`. Maybe the component should incorporate it.

Reply:
Color text_secondary is for the table views, while mobile views don't usually have this color
and I would like to keep only logics in the component